apps/prairielearn/assets/scripts/instructorAssessmentManualGradingInstanceQuestion.ts

```diff
@@ -63,62 +71,70 @@ function resetInstructorGradingPanel() {
+          .forEach((toggle) => {
+            toggle.checked = use_percentage;
+          });
+        document.querySelectorAll<HTMLElement>('.js-manual-grading-points').forEach((element) => {
+          element.style.display = use_percentage ? 'none' : '';
+        });
+        document
+          .querySelectorAll<HTMLElement>('.js-manual-grading-percentage')
+          .forEach((element) => {
+            element.style.display = use_percentage ? '' : 'none';
+          });
+        window.localStorage.manual_grading_score_use = use_percentage ? 'percentage' : 'points';
+        updatePointsView(null);
       });
-      window.localStorage.manual_grading_score_use = use_percentage ? 'percentage' : 'points';
-      updatePointsView(null);
+      toggle.checked = window.localStorage.manual_grading_score_use === 'percentage';
+      toggle.dispatchEvent(new Event('change'));
     });
-    toggle.checked = window.localStorage.manual_grading_score_use === 'percentage';
-    toggle.dispatchEvent(new Event('change'));
-  });
 
   // Auto points are disabled by default to avoid confusion, since they are not typically changed by this interface.
-  document.querySelectorAll('.js-enable-auto-score-edit').forEach((pencil) => {
+  document.querySelectorAll<HTMLElement>('.js-enable-auto-score-edit').forEach((pencil) => {
     pencil.addEventListener('click', function () {
       const form = this.closest('form');
-      form.querySelectorAll('.js-auto-score-value-info').forEach((element) => {
+      form?.querySelectorAll<HTMLElement>('.js-auto-score-value-info').forEach((element) => {
         element.style.display = 'none';
       });
-      form.querySelectorAll('.js-auto-score-value-input').forEach((input) => {
+      form?.querySelectorAll<HTMLElement>('.js-auto-score-value-input').forEach((input) => {
         input.classList.remove('d-none');
         input.style.display = '';
         input.querySelector('input')?.focus();
       });
     });
   });
 
-  document.querySelectorAll('.js-submission-feedback').forEach((input) => {
+  document.querySelectorAll<HTMLTextAreaElement>('.js-submission-feedback').forEach((input) => {
     input.addEventListener('input', () => adjustHeightFromContent(input));
     adjustHeightFromContent(input);
   });
 
-  document.querySelectorAll('.js-show-rubric-settings-button').forEach((button) =>
-    button.addEventListener('click', function () {
-      $('.js-rubric-settings-modal').modal('show');
-    }),
-  );
+  document
+    .querySelectorAll<HTMLButtonElement>('.js-show-rubric-settings-button')
+    .forEach((button) =>
+      button.addEventListener('click', function () {
+        $('.js-rubric-settings-modal').modal('show');
+      }),
+    );
 
   document
-    .querySelectorAll('.js-selectable-rubric-item')
+    .querySelectorAll<HTMLInputElement>('.js-selectable-rubric-item')
     .forEach((item) => item.addEventListener('change', computePointsFromRubric));
```

Comment:
this call signature doesn't line up. I could not get the typechecker to be happy in this situation -- My understanding is this is trying to call `computePointsFromRubric(htmlInputElement, target)`. But the function signature will not accept that. Instead it calls  `computePointsFromRubric(target)`, which causes issues, as target is not a `HTMLInputElement`. @nwalters512 perhaps you can shed some insight as to the intention of these two lines?